### PR TITLE
UPD [privx] upgrade privx to 31.2

### PR DESCRIPTION
**Pull request by ssh-morty**
I have tested the upgrade, rollback, and installation for privx-kube.31.2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# **BREAKING CHANGES IN PRIVX HELM CHART 31.X.X**
+# **BREAKING CHANGES IN PRIVX HELM CHART 31.0.0**
```
Comment (izareenssh) [inline README.md:1]: Can you please change this back to X.X

File: charts/privx/migrations/31/README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ The backup will be created under the volume that was mounted as the claim
 `privx-backup-claim`. The backup folder will have the following naming
 structure:
 
-`privx-backup-PPPPP_YYYY-MM-DD-hhmm_30.X.X`
+`privx-backup-PPPPP_YYYY-MM-DD-hhmm_30.1.0`
```
Comment (izareenssh) [inline charts/privx/migrations/31/README.md:42]: Same here.

File: charts/privx/migrations/31/README.md [hunk 3]
```diff
@@ -82,7 +82,7 @@ helm upgrade --history-max 0 \
 
 3. Replace the placeholder backup folder name for the environment variable
 `BACKUP_DIR` in the file [restore.yaml](../restore.yaml) with the one from
-when the backup was taken (`privx-backup-PPPPP_YYYY-MM-DD-hhmm_30.X.X`). Make
+when the backup was taken (`privx-backup-PPPPP_YYYY-MM-DD-hhmm_30.1.0`). Make
```
Comment (izareenssh) [inline charts/privx/migrations/31/README.md:85]: Here as well

File: charts/privx/migrations/31/README.md (hunk 2 above)
Comment (ssh-morty) [reply charts/privx/migrations/31/README.md]: fix

File: README.md (hunk 1 above)
Comment (ssh-morty) [reply README.md]: fix

**Review by izareenssh — APPROVED** [review]

**Pull request merged**